mdjs-layout/index.js

```diff
@@ -1 +1,3 @@
-export * from './src/MdjsLayout.js';
+export const breakpoints = {
+  xl: '1030px',

```

Comment:
this is the part which was always confusing to me when it comes to breakpoints, because to me it's not too logical to look at standard dimensions (which means you try to target common devices) and instead more logical look at what makes sense for certain widths (fully decouple from devices and think design wise, change browser width and try to see where it makes sense to start changing rules for your specific layout)

in this case this value was derived from max content width + left navigation width and turned out to be this number

if I go from the other side and set it to 1024, I should then derive one of those 2 numbers from it, which is also fine, becuase those numbers are quite subjective too and don't need to be precise

it's funny how popular ranges correlate (or not correlate) to actual popular devices - https://gs.statcounter.com/screen-resolution-stats
I was looking at this data many times in my life and always thought that it's weird to base anything on precise numbers from it (like 1024), because you need to approximate this somehow due to a lot of close variants, not to speak about that popular variants are chaning over time

Reply:
I think to be able to base ranges on statictis, it needs to be more precise and demonstrated on a different graph
- a distrubution graph instead of a trend over time
- forget about widthXheight, use actual viewport size (so that even the same person with same device is counted properly depending on how exactly he has the browser open right now and in which mode: vertical or horizontal)

I never saw such a graph and not sure if anyone is collecting the numbers which could be used to make it
once I tried to find this info in Google Analytics for one of the projects which I worked on and needed to create a set of breakpoints for and turned out even GA doesn't collect viewport sizes... I must be missing smth here

Reply:
Hm that's fair, I'm not against moving against the paradigm of generic breakpoints and focusing more on what breakpoints make sense for an app based on its content. I think a combination of the two is probably best, I agree we'd need to have some more data on the "hot spots" across viewport dimensions and what kind of devices sit where in order to find "the correct breakpoint"

But again, not a blocker for this PR, current state is fine by me